playbooks/containerd-build/run.yaml

```diff
@@ -0,0 +1,29 @@
+- hosts: all
+  become: yes
+  roles:
+    - role: config-golang
+      arch: arm64
+  tasks:
+    - name: Build containerd
+      shell:
+        cmd: |
+          set -xe
+          apt-get update
+          apt-get install -y btrfs-tools libseccomp-dev git pkg-config
+          echo $PATH
```

Comment:
do not need these `echo ...` statements. as we have `set -x`, all the scripts running trace will be print out

Reply:
+1

Reply:
done